.github/workflows/qat.yml

```diff
@@ -16,47 +16,62 @@ on:
 
     secrets:
       BITBUCKET_READ_ONLY_SSH_KEY:
-        required: true
+        required: false
       GITHUB_READ_ONLY_SSH_KEY:
+        required: false
+      CHECKLY_API_KEY:
+        required: true
+      CHECKLY_ACCOUNT_ID:
         required: true
 
 permissions:
   contents: read
 
+# Set the necessary credentials and export variables we can use to instrument our test run. Use the ENVIRONMENT_URL
+# to run your checks against staging, preview or production.
+# See https://www.checklyhq.com/docs/cli/command-line-reference/#npx-checkly-deploy and supporting documentation for
+# more details on Checkly environment variables.
+env:
+  NPM_QAT_SCRIPT: "qat"
+  FEATURE_BRANCH_NAME: $(echo "${{ github.head_ref || github.ref_name }}" | sed -e 's/feature\/\(.*\)/\1/' | tr '[:upper:]' '[:lower:]')
+  CHECKLY_API_KEY: ${{ secrets.CHECKLY_API_KEY }}
+  CHECKLY_ACCOUNT_ID: ${{ secrets.CHECKLY_ACCOUNT_ID }}
+  ENVIRONMENT_URL: ${{ inputs.QA_SITE_DOMAIN }}
+  CHECKLY_TEST_ENVIRONMENT: ${{ inputs.QA_SITE_DOMAIN }}
+  CHECKLY_REPO_BRANCH: ${{ github.head_ref }}
+  CHECKLY_REPO_COMMIT_OWNER: ${{ github.actor }}
+  CHECKLY_REPO_SHA: ${{ github.sha }}
```

Comment:
Nitpick, since the repo sha is not part of checkly, perhaps just `COMMIT_SHA` or similar?

Reply:
It is a Checkly specific environment variable documented here: https://www.checklyhq.com/docs/cli/command-line-reference/
![Screen Shot 2023-10-09 at 12 52 58 PM](https://github.com/penske-media-corp/github-workflows-wordpress/assets/20386229/71196118-1395-4826-864f-ea400a2c06ed)

Reply:
I see how you're using this.. `-e CHECKLY_REPO_SHA=${{ env.CHECKLY_REPO_SHA }}` perhaps no need for an ENV? Just `-e CHECKLY_REPO_SHA=${{ github.sha }}` ?

Reply:
Ah ok.. I would say just try to avoid duplication of things.

Reply:
ah good call! I removed this duplication. Checkly is already getting the SHA in the env vars above.